packages/legacy/babel/static/plugins/inject-package-version.js

```diff
@@ -0,0 +1,55 @@
+/*!
+ * Copyright (c) 2015-2020 Cisco Systems, Inc. See LICENSE file.
+ */
+
+const t = require('@babel/types');
+
+const {version} = require('../../package.json');
+
+/**
+ * Simple babel transform for ensuring that every WebexPlugin (and WebexCore)
+ * have the correct version property
+ * @returns {Object}
+ */
+module.exports = function injectPackageVersion() {
+  return {
+    visitor: {
+      /**
+       * Adds a version property to all SparkPlugins
+       * @param {Object} path
+       * @returns {void}
+       */
+      CallExpression(path) {
```

Comment:
when there is a release for each package it points to the path

Reply:
This can be updated as a part of the `build` task in JIRA. This config was placed here specifically for ESLint to use.